crypto/dilithium/pqcrystals_dilithium_ref_common/poly.c

```diff
@@ -533,6 +484,9 @@ void poly_challenge(ml_dsa_params *params, poly *c, const uint8_t *seed) {
     c->coeffs[b] = 1 - 2*(signs & 1);
     signs >>= 1;
   }
+  /* FIPS 204. Section 3.6.3 Destruction of intermediate values. */
```

Comment:
Missing some integer cleanses here.

Reply:
added in [c33b520](https://github.com/aws/aws-lc/pull/2017/commits/c33b520969414e82fa7fca5628f5c7af8f79195b)

Reply:
The `signs` variable contains data rather than simply being an offset.